app/controllers/auth_controller.rb

```diff
@@ -16,14 +16,15 @@ def authorize
   def token
     params = request.parameters
     if params[:code] == Rails.application.config.auth_code
+      ttl = 1.hour
```

Comment:
If we're bothering to improve the readability of `3600`, I think we should use an unabbreviated variable name.